### PR TITLE
Add NATS subscriber

### PR DESCRIPTION
**Pull request by parsaaes**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -25,7 +26,9 @@ type (
 
 	// NATS configuration.
 	NATS struct {
-		URL string `koanf:"url"`
+		URL           string        `koanf:"url"`
```
Comment (1995parham) [inline config/config.go:29]: Awesome :see_no_evil:

File: config/config.go [hunk 2]
```diff
@@ -41,7 +44,7 @@ func New() Config {
 	k := koanf.New(".")
 
 	// load default configuration from file
-	if err := k.Load(structs.Provider(Default(), "konaf"), nil); err != nil {
+	if err := k.Load(structs.Provider(Default(), "koanf"), nil); err != nil {
```
Comment (1995parham) [inline config/config.go:47]: Thanks for correcting this :+1:

File: config/default.go [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
 	return Config{
 		NATS: NATS{
-			URL: "nats://127.0.0.1:4222",
+			URL:           "nats://127.0.0.1:4222",
+			MaxReconnect:  60,
+			ReconnectWait: 1 * time.Second,
 		},
 		Streaming: Streaming{
-			ClusterID: "4lie",
+			ClusterID: "charlie",
```
Comment (1995parham) [inline config/default.go:15]: @parsaaes Please fix this also on `.drone.yml`

**Review by 1995parham — CHANGES_REQUESTED** [review]

File: config/default.go (hunk 3 above)
Comment (parsaaes) [reply config/default.go:15]: Done.

**Review by 1995parham — APPROVED** [review]

**Pull request merged**